features/FEATURE_BLE/targets/TARGET_CORDIO/stack/ble-host/sources/stack/att/atts_sign.c

```diff
@@ -163,69 +160,53 @@ static void attsProcSignedWrite(attCcb_t *pCcb, uint16_t len, uint8_t *pPacket)
   /* find attribute */
   if ((pAttr = attsFindByHandle(handle, &pGroup)) != NULL)
   {
-    /* verify signed write is permitted */
-    if ((pAttr->settings & ATTS_SET_ALLOW_SIGNED) == 0)
+    /* verify permissions */
+    if (attsPermissions(pCcb->connId, ATTS_PERMIT_WRITE, handle, pAttr->permissions) != ATT_SUCCESS)
```

Comment:
I remember we made changes here as the check from the stack were incorrect. We should double  check if they are correct now